src/Spout/Writer/ODS/Internal/Worksheet.php

```diff
@@ -191,27 +192,33 @@ protected function getCellXML($cellValue, $styleIndex, $numTimesValueRepeated)
             $data .= ' table:number-columns-repeated="' . $numTimesValueRepeated . '"';
         }
 
-        if (CellHelper::isNonEmptyString($cellValue)) {
+        if ($cellValue instanceof Cell) {
```

Comment:
Mmmh. I could move it up in the dedicated writers - but it would still be duplicated. XLSX and ODS do some magic on their own when iterating over the "rows" array. Maybe a future refactoring step?